scripts/mcptam_install.bash

```diff
@@ -81,35 +97,34 @@ install_dependencies()
 	install_toon
 	install_libcvd
 	install_gvars3
-	rm -rf build_deps
 	cd ..
+	rm -rf build_deps
+
 }
 
 build_mcptam()
 {
-	# setup ros env
-	source $ROS_BASH
-
-	# create catkin workspace
-	mkdir -p $HOME/catkin_ws/src
-	cd $HOME/catkin_ws/src
-	catkin_init_workspace
-	cd -
-
-	# setup catkin workspace
-	cd $HOME/catkin_ws/
-	catkin_make
-	source devel/setup.bash
-	cd -
-
-	# copy mcptam to catkin workspace
-	cd ..
-	cp -R mcptam $HOME/catkin_ws/src
-	cd $HOME/catkin_ws/
-	catkin_make
+
+	if [ "$CI" = true ]; then
+		# grab pull request version to be tested, build done by travis
+		cd ..
+	    cp -R mcptam $CATKIN_WS/src
+	    cd -
+	else
+		# clone mcptam to catkin workspace and build for user
+		cd $CATKIN_WS/src
+		git clone https://github.com/wavelab/mcptam
```

Comment:
The idea is no, the instructions say just run this script and all will be good.  If you want to do it yourself, follow the step by step instructions on another page.